AntistasiOfficial.Altis/Missions/LOG_Medical.sqf

```diff
@@ -69,30 +69,37 @@ _sboxempty = createVehicle [_tipoVeh, _poscrash, [], 0, "CAN_COLLIDE"];
 //{_sboxempty lockCargo [_x, false];} forEach [0 ,1];
 //_sboxempty setFuel 0;
 
-_crate1 = "Land_PaperBox_01_small_closed_white_med_F" createVehicle _poscrash; // This code require an optimization, they are all same items which should spawn in different position and all deleted at same time later.
-_crate2 = "Land_PaperBox_01_small_closed_white_med_F" createVehicle _poscrash;
-_crate3 = "Land_PaperBox_01_small_closed_white_med_F" createVehicle _poscrash;
-_crate4 = "Land_PaperBox_01_small_closed_white_med_F" createVehicle _poscrash;
-_crate5 = "Land_PaperBox_01_small_closed_white_med_F" createVehicle _poscrash;
-_crate6 = "Land_PaperBox_01_small_closed_white_med_F" createVehicle _poscrash;
-_crate7 = "Land_PaperBox_01_small_closed_white_med_F" createVehicle _poscrash;
-_crate8 = "Land_PaperBox_01_small_closed_white_med_F" createVehicle _poscrash;
-_crate1 setPos ([getPos _sboxempty, 6, 185] call BIS_Fnc_relPos);
-_crate2 setPos ([getPos _sboxempty, 4, 167] call BIS_Fnc_relPos);
-_crate3 setPos ([getPos _sboxempty, 8, 105] call BIS_Fnc_relPos);
-_crate4 setPos ([getPos _sboxempty, 5, 215] call BIS_Fnc_relPos);
-_crate5 setPos ([getPos _sboxempty, 6, 125] call BIS_Fnc_relPos);
-_crate6 setPos ([getPos _sboxempty, 4, 147] call BIS_Fnc_relPos);
-_crate7 setPos ([getPos _sboxempty, 8, 82] call BIS_Fnc_relPos);
-_crate8 setPos ([getPos _sboxempty, 5, 222] call BIS_Fnc_relPos);
-_crate1 setDir (getDir _sboxempty + (floor random 180));
-_crate2 setDir (getDir _sboxempty + (floor random 180));
-_crate3 setDir (getDir _sboxempty + (floor random 180));
-_crate4 setDir (getDir _sboxempty + (floor random 180));
-_crate5 setDir (getDir _sboxempty + (floor random 180));
-_crate6 setDir (getDir _sboxempty + (floor random 180));
-_crate7 setDir (getDir _sboxempty + (floor random 180));
-_crate8 setDir (getDir _sboxempty + (floor random 180));
+
+// number of crate offsets = number of spawned crates
+_crateOffsets = [
+	[ 6, 185],
+	[ 4, 167],
+	[ 8, 105],
+	[ 5, 215],
+	[ 6, 125],
+	[ 4, 147],
+	[ 8, 82 ],
+	[ 5, 222]
+];
+
+_crates = [];
+{
+	_x params ["_distance","_heading"];
+	
+	_currentCrate =  "Land_PaperBox_01_small_closed_white_med_F" createVehicle _poscrash;
+	_currentCrate setPos ([getPos _sboxempty, _distance, _heading] call BIS_Fnc_relPos);
+	_currentCrate setDir (getDir _sboxempty + (floor random 180));
+
+	if (isClass(configfile >> "CfgPatches" >> "ace_dragging")) then { // check if ace dragging module is active
```

Comment:
That check will work (until ACE changes its internals)

What about checking ACE modules with provided `ace_common_fnc_isModLoaded`
https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_isModLoaded.sqf

See https://github.com/A3Antistasi/antistasiofficial/blob/master/AntistasiOfficial.Altis/Functions/fn_detectACE.sqf

Reply:
```
if(activeACE && {["ace_dragging"] call ace_common_fnc_isModLoaded}) then {
...
};
```

Reply:
But you don't have the function if ace is not loaded !? So what's the point of it ?